client.go

```
@@ -343,9 +362,13 @@ func (cl *Client) SetBaseURL(urlStr string, passThrough bool) error {
 	return err
 }
 
+// EnableASK instructs client to use ASK
+func (cl *Client) EnableASK() {
+	log.Debug("[EnableASK]::Enabling ASK ...")
+	cl.accountSwitchEnabled = true
+}
+
 // prepareURL returns URL which is used to make API call
-//
-// client
 func prepareURL(url *url.URL, path string) (*url.URL, error) {
 
 	rel, err := url.Parse(strings.TrimPrefix(path, "/"))
```

error is not returned if appeared